docs/formats.md

```diff
@@ -1,11 +1,10 @@
 # Formats
 
-Formats are one of the ways to create files that act as interfaces for your style dictionary. For example, you want to be able to
-use your style dictionary in CSS. You can use the `css/variables` template which will create a CSS file with variables from
+Formats are define the output of your created files for your style dictionary. For example, you want to be able to
```

Comment:
"are define"

Reply:
Thanks for pointing this out. Tried to go over everything one more time and make sure it was all up to date and easy to read.